building/build-debs/homeworld-admin-tools/src/configuration.py

```diff
@@ -164,6 +164,27 @@ def get_node(self, node_name: str) -> Node:
                 return node
         command.fail("no such node: %s" % node_name)
 
+    def has_hostname(self, hostname: str) -> bool:
+        if hostname.endswith(self.external_domain):
+            # strip external domain
+            hostname = hostname[:-(len(self.external_domain) + 1)]
+        elif hostname.endswith(self.internal_domain):
+            # strip internal domain
+            hostname = hostname[:-(len(self.internal_domain) + 1)]
+        return self.has_node(hostname)
```

Comment:
It doesn't seem like this is used anywhere, so it should probably be removed, especially since it seems to duplicate code.